source/Missions/Services/Arena/RandomEquipmentGenerator.cs

```diff
@@ -43,20 +45,28 @@ internal class RandomEquipmentGenerator : IRandomEquipmentGenerator
         {
             IEnumerable<ItemObject> allItems = Game.Current.ObjectManager.GetObjectTypeList<ItemObject>();
             IDictionary<ItemTypeEnum, List<ItemObject>> result = new Dictionary<ItemTypeEnum, List<ItemObject>>();
+            //Here im harcoding the elements that are causing the arrows not to spawn and the same for the throwing weapons
+            HashSet<string> deleteItems = new HashSet<string>{"ballista_projectile_burning", "ballista_projectile", "throwing_stone", "boulder",
```

Comment:
move this to the class scope `private static readonly...`
Rename this to `ExcludedItems`

Should be perfect after that

Reply:
ye my bad, i forgot